kernel/dw1000.c

```diff
@@ -2748,6 +2808,44 @@ static int dw1000_reset(struct dw1000 *dw)
 	return 0;
 }
 
+/**
+ * dw1000_load_xtalt() - Load XTALT calibration value
+ *
+ * @dw:			DW1000 device
+ * @return:		0 on success or -errno
+ */
+static int dw1000_load_xtalt(struct dw1000 *dw)
+{
+	uint32_t value;
+	uint8_t xtalt;
+	int rc;
+
+	/* Default value for XTALT */
+	xtalt = DW1000_FS_XTALT_XTALT_MIDPOINT;
+
+	/* Read XTAL_Trim calibration value from OTP */
+	if ((rc = regmap_read(dw->otp, DW1000_OTP_REV_XTALT, &value)) != 0)
+		return rc;
+	if (value != ~0)
```

Comment:
Can this be an explicit 0xffffffff, to cope with the possibility that "int" is not 32 bits?